### PR TITLE
My branch

### PR DESCRIPTION
**Pull request by yogeshpaliyal**

### CODE REVIEW

File: src/Pages/Librarylist.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+                  library having Ac, Wifi etc.
+                </p>
+                <Button onClick={() => openModal(library)}>Book Now</Button>
+              </div>
+            </div>
+          ))}
+        </div>
+      </div>
+      {isModalOpen && selectLibrary && (
+        <div className="fixed inset-0 flex items-center justify-center bg-black bg-opacity-50">
```
Comment (yogeshpaliyal) [inline src/Pages/Librarylist.tsx:84]: Create a new component for modal

File: src/Pages/Librarylist.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+import { Button } from "@/components/ui/button";
+import Nav from "@/mycomponents/Nav";
+
+function Librarylist() {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [isSecondModalOpen, setIsSecondModalOpen] = useState(false);
+
+  const [selectLibrary, setSelectLibrary] = useState<Library | null>(null);
+
+  interface Library {
```
Comment (yogeshpaliyal) [inline src/Pages/Librarylist.tsx:12]: Move it out of component

File: src/Pages/Librarylist.tsx (hunk 1 above)
Comment (UmeshPaliyaal) [reply src/Pages/Librarylist.tsx]: resolved

File: src/Pages/Librarylist.tsx (hunk 2 above)
Comment (UmeshPaliyaal) [reply src/Pages/Librarylist.tsx:13]: resolved

File: package.json [hunk 3]
```diff
@@ -10,13 +10,17 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@auth0/auth0-react": "^2.2.4",
```
Comment (yogeshpaliyal) [inline package.json:13]: Remove this

File: package.json [hunk 4]
```diff
@@ -10,13 +10,17 @@
...
+    "@radix-ui/react-checkbox": "1.1.1",
     "@radix-ui/react-slot": "1.1.0",
     "@reduxjs/toolkit": "2.2.6",
     "class-variance-authority": "0.7.0",
     "clsx": "2.1.1",
+    "jwt-decode": "4.0.0",
```
Comment (yogeshpaliyal) [inline package.json:19]: Remove this

File: package.json (hunk 3 above)
Comment (UmeshPaliyaal) [reply package.json]: Deleted

File: package.json (hunk 4 above)
Comment (UmeshPaliyaal) [reply package.json]: Deleted . yet failed

**Pull request merged**